tests/Microsoft.Identity.Test.Unit/WwwAuthenticateParametersTests.cs

```diff
@@ -193,48 +262,57 @@ public async Task CreateFromResourceResponseAsync_HttpClient_Null_Async(HttpClie
         public async Task CreateFromResourceResponseAsync_HttpClientAsync()
         {
             const string resourceUri = "https://example.com/";
+            var tenantId = Guid.NewGuid().ToString();
 
             var handler = new MockHttpMessageHandler
             {
                 ExpectedMethod = HttpMethod.Get,
                 ExpectedUrl = resourceUri,
-                ResponseMessage = CreateInvalidTokenHttpErrorResponse()
+                ResponseMessage = CreateInvalidTokenHttpErrorResponse(tenantId)
             };
             var httpClient = new HttpClient(handler);
 
-            var _ = await WwwAuthenticateParameters.CreateFromResourceResponseAsync(httpClient, resourceUri).ConfigureAwait(false);
+            _ = await WwwAuthenticateParameters.CreateFromResourceResponseAsync(httpClient, resourceUri).ConfigureAwait(false);
         }
 
         [TestMethod]
         public async Task CreateFromResourceResponseAsync_HttpClient_CancellationToken_Async()
         {
             const string resourceUri = "https://example.com/";
+            var tenantId = Guid.NewGuid().ToString();
 
             var handler = new MockHttpMessageHandler
             {
                 ExpectedMethod = HttpMethod.Get,
                 ExpectedUrl = resourceUri,
-                ResponseMessage = CreateInvalidTokenHttpErrorResponse()
+                ResponseMessage = CreateInvalidTokenHttpErrorResponse(tenantId)
             };
             var httpClient = Substitute.For<HttpClient>(handler);
 
             var cts = new CancellationTokenSource();
 
-            var _ = await WwwAuthenticateParameters.CreateFromResourceResponseAsync(httpClient, resourceUri, cts.Token).ConfigureAwait(false);
+            _ = await WwwAuthenticateParameters.CreateFromResourceResponseAsync(httpClient, resourceUri, cts.Token).ConfigureAwait(false);
 
             httpClient.Received();
         }
 
         [DataRow(null)]
         [DataRow("")]
         [TestMethod]
-        public async Task CreateFromResourceResponseAsync_ResourceUri_Async(string resourceUri)
+        public async Task CreateFromResourceResponseAsync_Incorrect_ResourceUri_Async(string resourceUri)
         {
             Func<Task> action = () => WwwAuthenticateParameters.CreateFromResourceResponseAsync(resourceUri);
 
             await Assert.ThrowsExceptionAsync<ArgumentNullException>(action).ConfigureAwait(false);
         }
 
+        public async Task CreateFromResourceResponseAsync_ResourceUri_Async(string resourceUri)

```

Comment:
Is this a test? Missing TestMethod attribute?

Reply:
Oh, good catch. Thanks! Fixed.